ddl/ddl_db_test.go

```diff
@@ -1376,7 +1380,7 @@ func (s *testDBSuite) testRenameTable(c *C, storeStr, sql string) {
 	failSQL = fmt.Sprintf(sql, "test1.t2", "test1.t2")
 	s.testErrorCode(c, failSQL, tmysql.ErrTableExists)
 
-	s.tk.MustExec("drop table test1.t2")
+	s.tk.MustExec("drop database test1")
```

Comment:
Why change this case?

Reply:
Because Other tests will create this database. So we need clean it.